jboss-kie-smartrouter/added/launch/jboss-kie-smartrouter.sh

```diff
@@ -142,6 +147,42 @@ function configure_controller_access {
     fi
 }
 
+function configure_router_tls() {
+    # If the path, alias, or password is empty exit early and skip https
+    if [ -z "${KIE_SERVER_ROUTER_TLS_KEYSTORE}" ] || \
+       [ -z "${KIE_SERVER_ROUTER_TLS_KEYSTORE_KEYALIAS}" ] || \
+       [ -z "${KIE_SERVER_ROUTER_TLS_KEYSTORE_PASSWORD}" ]; then
+        echo Missing value for TLS keystore path, alias, or password, skipping https setup
+	return
+    fi
+
+    # Allow for optional volume mount or empty secret
+    if ! [ -f "${KIE_SERVER_ROUTER_TLS_KEYSTORE}" ]; then
+	echo Keystore file "${KIE_SERVER_ROUTER_TLS_KEYSTORE}" not found or not a regular file, skipping https setup
+	return
+    fi
+
+    # If the keystore is not readable, smartrouter startup will throw an exception
+    # resulting in the http port being unavailable as well. So make sure ...
+    keytool -list -alias ${KIE_SERVER_ROUTER_TLS_KEYSTORE_KEYALIAS} \
+	          -storepass ${KIE_SERVER_ROUTER_TLS_KEYSTORE_PASSWORD} \
+	          -keystore ${KIE_SERVER_ROUTER_TLS_KEYSTORE} &> /dev/null
+    if [ "$?" -ne 0 ]; then
+	echo Unable to read TLS keystore, skipping https setup
+	return
+    fi
+
+    JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.router.tls.keystore=${KIE_SERVER_ROUTER_TLS_KEYSTORE}"
+    JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.router.tls.keystore.keyalias=${KIE_SERVER_ROUTER_TLS_KEYSTORE_KEYALIAS}"
+    JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.server.router.tls.keystore.password=${KIE_SERVER_ROUTER_TLS_KEYSTORE_PASSWORD}"
+
+    local kieServerRouterPortTLS="${KIE_SERVER_ROUTER_PORT_TLS}"
+    if [ "${kieServerRouterPortTLS}" = "" ]; then
+        kieServerRouterPort="9443"
```

Comment:
Should "kieServerRouterPort" actually be "kieServerRouterPortTLS"?

Reply:
yes, it should :) cut and paste damage I'm guessing